apps/tenant-manager-cli/src/Infrastructure/Operations/CompanyCreatorAdapter.php

```
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Operations;
+
+use Nexus\TenantOperations\Contracts\CompanyCreatorAdapterInterface;
+
+final readonly class CompanyCreatorAdapter implements CompanyCreatorAdapterInterface
+{
+    public function createDefaultStructure(string $tenantId, string $companyName): string
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$tenantId'. (undefined)

(UnusedFormalParameter)